### DISCUSSION TITLE
I'm trying to get videos based on a used hashtag.

### DISCUSSION THREAD

**Discussion by sniGithub2022** [davidteather/TikTok-Api#936] [Q&A]
when I use api.hashtag, I can get the tag information ( {'videoCount': 13300, 'viewCount': 274800000}  exc.) but when I try to reach the videos I find none. I used metoo and restorethesnyderverse as the searched tags so there must be many videos.
can someone please help me?

the Code:

with TikTokApi() as api:
    tag = api.hashtag(name="metoo")
    print(tag.info())
    for video in tag.videos():
        print(video.id)

**Comment by davidteather** [comment]
Hashtag videos should be fixed in V6
